### ISSUE TITLE
Java/scala models don't compile when importing message directly

### ISSUE THREAD

**Issue by ligadata-william** [LigaData/Kamanja#772]
Given a message named 'com.ligadata.automation.tests.rdd.messagebytearray.MessageByteArrayTestMsg', if I include the line
```
import com.ligadata.automation.tests.rdd.messagebytearray.MessageByteArrayTestMsg
```
in my java/scala model (in this case scala), MetadataAPI fails to compile the model that is importing the message with the error:
object MessageByteArrayTestMsg does not exist in package import com.ligadata.automation.tests.rdd.messagebytearray

As soon as I changed the import statement to 
```
import com.ligadata.automation.tests.rdd.messagebytearray._
```
for scala, MetadataAPI was able to compile the model.

This appears to be happening because during the initial compilation phase where it looks for a message with a V# as part of the package, it doesn't see the object in the package because it only sees an additional namespace of V#, which is added automatically.

If the import statement already exists at this phase, we should replace it with the proper import during compilation.

**Comment by ligadata-yasser** [comment]
@ligadata-william 
fixed, and issued a pull request #1157 . please verify

**Comment by ligadata-william** [comment]
Verified as fixed. Closing.
